### ISSUE TITLE
[Accessibility] [Alert > 4th Sub-component]: Color Contrast ratio is not sufficient for the focus outline on 'Linked text'.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2089]
[A11Y-550](https://jira.cengage.com/browse/A11Y-550) _Orion is reviewing_
[A11Y-551](https://jira.cengage.com/browse/A11Y-551) ✅
[A11Y-552](https://jira.cengage.com/browse/A11Y-552) ✅ 
[A11Y-553](https://jira.cengage.com/browse/A11Y-553) ✅ 

**Issue 1:**
**[Accessibility] [Alert > 4th Sub-component]: Color Contrast ratio is not sufficient for the focus outline on 'Linked text'.**

Platform:
Windows 11 / Chrome

Steps to reproduce:
1. Open the URL: https://react-magma.cengage.com/version/4.9.1/
2. Now Navigate to side navigation and expand 'Components' button.
3. Now activate 'Alert' link under API and verify.

Actual Result:
It is observed that the focus outline applied to 'Linked text' link has a low color contrast against the background. The contrast ratio does not meet the minimum requirement of 3:1, making the focus indicator difficult to detect.

Expected Result:
The expected is that the focus outline on 'Linked text' link should have a minimum contrast ratio of 3:1 against the surrounding background to ensure that users can clearly perceive which element is focused.

Instances:

1. [Accessibility][Badge > Event Handling ]: It is observed that the focus outline applied to 'Clicked mee too' link has a low color contrast against the background. The contrast ratio does not meet the minimum requirement of 3:1, making the focus indicator difficult to detect.
https://app.screencast.com/H3h4BoKlcj4fi

WCAG Guideline:
1.4.11 Non-text Contrast Level AA

User Impact:
Low-vision users and keyboard-only users may not be able to see where the focus is, making navigation difficult or impossible. This creates usability barriers and fails to support users who depend on visible focus indicators.

Recommendation:
Update the CSS to apply a high-contrast color to focus outlines that meets or exceeds the 3:1 contrast ratio requirement.

Screencast:
https://app.screencast.com/D82tPY68OJFyu

Team note: We need to be using the inverse focus color on the links in the alerts, and we need a 2px offset on those clickable badges, the same we do on buttons.

**Comment by silvalaura** [comment]
Our example uses `<a href="#">linked text</a>` instead of the hyperlink component with isInverse, so we can update this example

**Comment by olehnoskov-cengage** [comment]
**Alert** -> **Basic Usage** (Example with Link)
<img width="716" height="498" alt="Image" src="https://github.com/user-attachments/assets/e8ec7185-a58b-461b-9aee-3b4e42e0fda9" />

**Alert** -> **Variants** (Example with Link)
<img width="716" height="498" alt="Image" src="https://github.com/user-attachments/assets/92740429-051d-4833-948a-baf3548014e5" />


**Alert** -> **Inverse** (Example with Link)
<img width="712" height="494" alt="Image" src="https://github.com/user-attachments/assets/f095ad01-43bd-4079-81f7-0dda007362a6" />

**Comment by olehnoskov-cengage** [comment]
@orion-cengage 
Could you verify that all our colors have been applied correctly and that no updates are needed?

**Comment by nikitaorliak-cengage** [comment]
Tested on:
- V5: https://rc--upbeat-sinoussi-f675aa.netlify.app/api/badge/#event_handling
- V4: https://next--upbeat-sinoussi-f675aa.netlify.app/api/badge/#event_handling

Everything works as expected. No bugs found.
